src/main/kotlin/eu/europa/ec/eudi/sdjwt/vc/SdJwtVcVerifier.kt

```diff
@@ -219,6 +221,40 @@ private suspend fun issuerJwkSource(
     }
 }
 
+/**
+ * [JWKSource] implementation for DID Documents.
+ *
+ * When [JWKSource.get] is invoked, it ignores the [JWKSelector] provided and instead one that matches all
+ * the properties of the provided [JWSHeader], but the Key ID.
+ */
+private class DIDJWKSet<C : SecurityContext>(jwsHeader: JWSHeader, val jwkSet: JWKSet) : JWKSource<C> {
```

Comment:
@dzarras Perhaps we should "nest" this class to the companion object of our processor, since this `JWKSource` can only be safely used from within this processor.

Reply:
Could you check now? Is that OK?

Reply:
Yes